### ISSUE TITLE
Design Threaded Discussions

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/action#3334]
tl;dr it is facebook's commenting system with the ability to turn a comment into a task and post something anonymously. Exploration in #3333. Drawing attention to new comments is out of scope for this.

### AC
- [ ] In the Discussion phase, each reflection has its own column
- [ ] Below each reflection is a 1-line text area with a placeholder that says "Write a comment"
- [ ] On the left side of the text area is the user avatar.
- [ ] Hovering over the user avatar puts a dark filter on the picture with a "?" overlap & a tooltip that says "Click to comment anonymously"
- [ ] Clicking the user avatar turns anonymity on & the avatar turns into a question mark.
- [ ] On the right side of the text area is an icon when hovered says "Create Task". When clicked, the input turns into a Task card & any text entered into the input is copied over
- [ ] When a comment or task is created, it appears below the reflection, but above "Add a comment" input, exactly like facebook. (forces the viewer to scroll to the bottom before they can create a new top-level task/comment)
- [ ] Once a task/comment is posted, There is a new text area with the placeholder "Write a reply"

### EE 13 pts

**Comment by jordanh** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/1274051) writes:

> [It's very important to us that we can make] comments on ideas and action items

**Comment by ackernaut** [comment]
I think the next steps here are (continuing [thinking](https://github.com/ParabolInc/action/issues/3344#issuecomment-571663164) from 1:1 UX research):
- Find the common and unique UX patterns in a discussion thread for the following cases: Retro topics, Check-In agenda items, 1:1 agenda items, async threads + decisions
- Gather sketches, concepts, and prototypes
- Gather feedback on the retro thread UX from users
- Scope a v1 of a retro group discussion thread implementation
- Kick off a 1:1 UX design issue

I think as an overall effort to improve retro and threading UX, we keep pushing on reactji and adding a vote count setting affordance to the lobby. Folks will be able to separate sentiment from ranking what they should talk about, and be able to add more comments other than just tasks. And we’re well on our way to adding threads in other meeting types.

**Comment by ackernaut** [comment]
Next steps here:
- Create a prototype showing variants of a v1 thread per comment only, per group only, and both comments and group
- Engage UX user testing list with sync and async feedback options

**Comment by ackernaut** [comment]
See #3545 for implementation details in the works
